setup.py

```diff
@@ -58,7 +58,7 @@ def get_version(rel_path):
     license="Apache License 2.0",
     install_requires=[
         "distinctipy",
-        "nmslib",
+        # "nmslib",
```

Comment:
Is this commented out because it won't build on your computer? I don't think we want to change this until we update the storage layer.

Reply:
Oops, my bad! That was supposed to be uncommented. NMSLib doesn't build on windows for python version >= 3.9, so I uncommented it and re-ran everything on a Python 3.8 kernel. Commenting it was for debugging purposes and is fixed now.